### ISSUE TITLE
Can't use keyboard to browse through levels or mods

### ISSUE THREAD

**Issue by albertz** [openlierox/openlierox#526]
Converted from [SourceForge issue 2960637](http://sourceforge.net/support/tracker.php?aid=2960637), submitted by sakmongkol

In earlier versions, it was possible to use a letter key in the mod and level lists to go to the first mod or level beginning with that letter. Now that all the mods and levels have the [LX] or [GUS] tag in front of them, you can't do that anymore.

**Comment by albertz** [comment]
Submitted by albertzeyer

Thanks for the report.

Several possible solutions:
1. We may ignore the "[...] " prefix for any keyboard input.
2. We put "[...]" as a postfix (at the end), not as a prefix.
3. We don't show it at all.

Any suggestions?

**Comment by albertz** [comment]
Submitted by dark_charlie

I would suggest showing an icon instead of the prefix. That will not break the key searching, will look better, will be less confusing (newbie: WTF is GUS?) and will provide the same information.

**Comment by albertz** [comment]
Submitted by albertzeyer

> newbie: WTF is GUS?

newbie: WTF is that icon about? :P

Or to put it different: Why would it cause less confusion? Or why would it be more clear to the user?

But I agree that it looks nicer.

About your suggestion: Should it be sorted then just by name? The reason I made it a prefix is because I _wanted_ to have it sorted first by the type, then by name.

Another problem: Wedon't have those icons and nobody can make them. (We don't have a designer in the team.) It probably would take a year or so until somebody else makes them for us (as it was with the flag icons - it took over a year until we got some gfx for them).
